### PR TITLE
Update to Rails 4.2

### PR DESCRIPTION
**Pull request by awead**
Includes changes to mime types. See the changes to mime types. I'm not sure why this happened, but it appears to be related to Rails. Do you all think this poses any issues? @mjgiarlo @cam156?

### CODE REVIEW

File: spec/models/generic_file_spec.rb [hunk 1]
```diff
@@ -145,7 +145,7 @@
 
     describe "an excel file" do
       it "should create derivatives" do
-        file_with_produced_web_and_thumbnail('test.xls', 'application/vnd.ms-excel')
+        file_with_produced_web_and_thumbnail('test.xls', 'application/excel')
```
Comment (mjgiarlo) [inline spec/models/generic_file_spec.rb]: This is a deprecated MIME type. We may need to investigate how we're using the mime-types gem to troubleshoot this.

File: spec/models/generic_file_spec.rb [hunk 2]
```diff
@@ -157,7 +157,7 @@
 
     describe "an powerpoint file" do
       it "should create derivatives" do
-        file_with_produced_access_and_thumbnail  'FlashPix.ppt',  'application/vnd.ms-powerpoint','application/pdf'
+        file_with_produced_access_and_thumbnail  'FlashPix.ppt',  'application/powerpoint','application/pdf'
```
Comment (mjgiarlo) [inline spec/models/generic_file_spec.rb]: This is a deprecated MIME type. We may need to investigate how we're using the mime-types gem to troubleshoot this.

File: spec/models/generic_file_spec.rb (hunk 1 above)
Comment (awead) [inline spec/models/generic_file_spec.rb]: We're using the current version, so I don't know why the deprecated one is showing up.

**Comment by awead** [comment]
This got the green light after today's standup, but an issue has been logged in case we need to revisit.

**Pull request merged**